sema/commons/j2/j2_functions.py

```
@@ -145,24 +190,24 @@ def xsd_format(content, type_name: str, quote: str = "'"):
 
-def regexreplace(find: str, replace: str, text: str):
+def regexreplace(find: str, replace: str, text: str) -> str:
     return re.sub(find, replace, text)
 
 
 class ValueMapper:
     def __init__(self):
         self._map = dict()
 
-    def add(self, key, val):
+    def add(self, key, val) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `key`

(ANN001)